client/galaxy/scripts/components/RuleCollectionBuilder.vue

```diff
@@ -300,11 +300,14 @@
         </div>
         <div class="rule-footer footer flex-row no-flex vertically-spaced" v-if="ruleView == 'source'">
             <option-buttons-div>
+                <button @click="attemptRulePreview" class="btn btn-secondary rule-btn-cancel">
```

Comment:
this should probably call a different function but I did not find a proper one

Reply:
This button does the same thing as the Apply button right?

Reply:
Yeah, I would like it to do proper cancel instead, hence my comment.

Reply:
I see - I'll see if I can make that work. Probably just reset following by apply.